chia/full_node/weight_proof.py

```diff
@@ -831,27 +835,6 @@ def handle_end_of_slot(
     )
 
 
-def compress_segments(full_segment_index, segments: List[SubEpochChallengeSegment]) -> List[SubEpochChallengeSegment]:
```

Comment:
Seems like you drop this because its unused (and not longer needed?) code, so imo this should be a separate PR rather instead of dropping it here to avoid the need of hinting it?

Reply:
yeah, probably. However, they were exposed as unused by the hinting, so in some sense it is "fixing" the hints